libs/watt/package/picker/datepicker/watt-datepicker.component.ts

```diff
@@ -186,16 +138,10 @@ export class WattDatepickerComponent extends WattPickerBase implements Validator
     })
   );
 
-  /**
-   * @ignore
-   */
   getPlaceholderByLocale(locale: WattSupportedLocales): string {
     return locale === 'da' ? 'dd-mm-åååå' : 'dd-mm-yyyy';
   }
 
-  /**
-   * @ignore
-   */
   getRangePlaceholder(): string {
     return this.placeholder + this.rangeSeparator + this.placeholder;
   }
```

Comment:
nitpick: the reason for having the ignore comments was mainly for Storybook, so ignored properties do not incorrectly show up as inputs in the "Controls" panel

<img width="1386" height="445" alt="Image" src="https://github.com/user-attachments/assets/ea964013-6eb9-4b22-82d0-7cd3ce6b2efb" />

Seems like in current Storybook version these comments do not matter anymore since the inputs are no longer shown in the panel.

<img width="1251" height="457" alt="Image" src="https://github.com/user-attachments/assets/b2741986-6c8b-461f-9360-dbd93e78879e" />

Reply:
That is why I removed them :-)